en/page-breaking.saty

```diff
@@ -0,0 +1,288 @@
+          as footnote.
+          Return value has type \type(Type.inline-boxes);.
+          Associate with a place inserted this return value, 
+          and calculate inserting the footnote at processing page-breaking.  
+          As inline-boxes feature itself is equivalent as \codem{inline-nil}.
+        }%
+
+      \val-explain?:(`hook-page-break`)({hook-page-break})%
+        (Type.(((record [({page-number}, int)] --> (point --> unit))) --> inline-boxes)) {
+          Use as \codem{hook-page-break \meta{f}}.
+          Return value is inline-boxes.
+          Features in typesetting is equivalent as \codem{inline-nil}.
+          But, a hooking function \codem{\meta{f}} starts
+          when fixed place including here is which page in processing page-breaking.
+          
+          The case mined multiple hooking functions at the same page,
+          the functions starts by page contents order from initial to last.
+          
+          where result of page-breaking, inline-boxes which mined *hook* function 
+          is included at page ${n}, and refer point is ${\tuple{|x|y|}} then,
+          \codem{\meta{f}} starts when it received 
+          \codem{(\| page-number = \meta{n}\; \|)} as 1st argument 
+          and 
+          \codem{(\meta{x}, \meta{y})} as 2nd argument.
+          You can invoke *side-effects* with these argument values
+          like to register cross-reference linking based on page number with \codem{\meta{f}} 's contents.
+      
+        }%
+
+      About thing “To evaluate graphics is triggerd at page-breaking” 
+      written at \ref-chapter(`graphics`); is 
+      as same as this \codem{hook-page-break}.
+      Functions that receives reffered point and returns graphics,
+      like 4th argument of \codem{inline-graphics}, and 
+      hooking functions by \codem{hook-page-break}
+      are start evaluations at the same time.
+    }
+  >
++section{Basis of classfile}<
+
+  +p{
+    You got matter about page-breaking process.
+    Next, you would get nice idea to create a classfile.
+    Then, you may notice until you have not think ever:\quad;
+    where is *Outermost* text processing context from?\quad;
+    This is much simple. You can receive by primitives like:
+
+    \val-explain?:(`get-initial-context`)({get-initial-context})%
+    (type.(length --> (cmdI [math] --> context))){
+      This Returns default text processing context.
+      1st argument as a paragraph width,
+      2nd argument as a command that applying to math equation appeard at inline-text
+      like \codem{set-math-command}.
+    }%
+    \codem{get-initial-context} is only one that able to set a paragraph width.
+      There is \codem{get-initial-context} in a classfile that work as well. 
+      So, transform text processing context returned by this as *outermost* text processing context by needs.
+  } 
+
+
+      +example{
+        A definition of “A classfile, one of minimum that seems to work well” is like this:
```

Comment:
A classfile, one of minimum that seems to work well
↓
an essentially minimum class file that is more or less able to be used